lib/ansible/modules/database/misc/redis.py

```diff
@@ -93,6 +93,14 @@
             - A redis config value.
         required: false
         default: null
+    config_write:
```

Comment:
I kinda agree with you, but at the same time I wanted to stay close to the redis terminology (`CONFIG REWRITE`). 
In my mind, `persistent` would somehow imply that only the changes made through the current ansible task are made persistent. However, the redis `CONFIG REWRITE` command will save the current running config, including the changes that this precise ansible task has made AND any other config changes potentially made by something/someone since the redis server was started.